apps/openchallenges/app/src/config/config.json

```diff
@@ -6,5 +6,6 @@
   "environment": "development",
   "googleTagManagerId": "",
   "ssrApiUrl": "http://openchallenges-api-gateway:8082/api/v1",
-  "enableOperationFilter": false
+  "enableOperationFilter": false,
+  "showAnnouncement": false
```

Comment:
The template in the same folder should be updated along with the corresponding environment variable in `.env.example`.